elcid/__init__.py

```diff
@@ -25,12 +25,13 @@ class Application(application.OpalApplication):
         'js/elcid/controllers/remove_patient_step.js',
 
         'js/elcid/services/demographics_search.js',
+        'js/elcid/services/demographics_search.js',
 
 
         'js/elcid/controllers/tagging_step.js',
         'js/elcid/controllers/investigations.js',
         # used in the blood culture forms
```

Comment:
I'm not sure the comment is useful when it's above exception_handler.js ....

Reply:
👍